src/document-util.ts

```diff
@@ -426,12 +426,13 @@ export function attachCommentsToFirstStatement(
   const firstStatement: (estree.Statement|estree.ModuleDeclaration)&RecastNode =
       statements[0]!;
 
-  const recastComments: RecastComment[] = comments.map((c) => {
+  const recastComments: RecastComment[] = comments.map((comment) => {
+    const escapedComment = comment.replace('*/', '*\\/');
```

Comment:
note that this only escapes the closing comment, leaving the opening comment untouched. This is syntactically all we need to fix the issue but for symmetry we may want to escape both. Open to thoughts here.

Reply:
+1